backend/src/main/java/gov/cdc/usds/simplereport/validators/FileRow.java

```diff
@@ -0,0 +1,10 @@
+package gov.cdc.usds.simplereport.validators;
+
+import gov.cdc.usds.simplereport.service.model.reportstream.FeedbackMessage;
+import java.util.List;
+
+public interface FileRow {
```

Comment:
nit: the naming of this is a little confusing, could it be something like `FileRowValidator`? same for the implementation files, too. It makes sense in the project structure, I just found skimming through I had to think twice about the name.

Reply:
The methods the interface defines right now are used for validation, but thats not necessarily what the class needs to be used for as seen in the [PatientBulkUploadService](https://github.com/CDCgov/prime-simplereport/blob/main/backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadService.java#L104).  I did notice that it was in the validators folder which may create confusion so I will move it to the model folder instead!

Reply:
Awesome! Yes, I think moving it out of the validator will clear up the confusion! 🌮